smt/applications/tests/test_ego.py

```diff
@@ -152,6 +194,48 @@ def test_rosenbrock_2D_SBO(self):
         self.assertTrue(np.allclose([[1, 1]], x_opt, atol=1))
         self.assertAlmostEqual(0.0, y_opt.item(), delta=1)
 
+    @unittest.skipIf(int(os.getenv("RUN_SLOW_TESTS", 0)) < 1, "too slow")
+    def test_function_ego_GPX_rosenbrock_2d(self):
```

Comment:
To be consistent with previous test names
```suggestion
    def test_rosenbrock_2D_GPX(self):
```